Test/ghost/AmbiguousGhost.dfy

```diff
@@ -0,0 +1,16 @@
+// RUN: %dafny /compile:0 /dprint:- "%s" /env:0 > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// Test that the `IsTypeSequence` method of the parser allows tuples with ghost components.
+module M1 {
+  class C1 {
```

Comment:
The `module` and `class` are allowed, but since they are not used in the example, you might as well remove them.

Reply:
Done.